classes/migration/install/JournalsMigration.php

```diff
@@ -29,7 +29,7 @@ public function up(): void
             $table->comment('A list of all journals in the installation of OJS.');
             $table->bigInteger('journal_id')->autoIncrement();
             $table->string('path', 32);
-            $table->float('seq', 8, 2)->default(0)->comment('Used to order lists of journals');
```

Comment:
[Motivation for this change here](https://laravel.com/docs/11.x/upgrade#floating-point-types). I'd rather just go to `->float('seq')` rather than specifying the magic number `53` -- at some point we can convert these to integers, but let's not do it in this PR.